src/core/components/files.js

```diff
@@ -120,6 +120,23 @@ module.exports = function files (self) {
     getPull: promisify((ipfsPath, callback) => {
       callback(null, exporter(ipfsPath, self._ipldResolver))
     })
```

Comment:
Looks like you forgot a comma at the end of this function. I think that's why the build failed.